From 54b3aee5a31b754ffef493424048dc7f6b52abaf Mon Sep 17 00:00:00 2001
From: Miguel Angel Nubla <miguelangel.nubla@gmail.com>
Date: Thu, 1 Jun 2023 02:09:39 +0200
Subject: [PATCH 1/2] Makefile downloader enable follow redirects

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index db76aad6c68..0a542e7d88c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -296,7 +296,7 @@ if test "x${WGET}" != "x:"; then
   DOWNLOAD_TO_STDOUT="${WGET} -q -O-"
   DOWNLOAD_TIMEOUT='--timeout=$1'
 elif test "x${CURL}" != "x:"; then
-  DOWNLOAD="${CURL} -O --progress-bar -w \"%{url_effective}\n\""
+  DOWNLOAD="${CURL} -L -O --progress-bar -w \"%{url_effective}\n\""
   DOWNLOAD_TO_STDOUT="${CURL} -Ls"
   DOWNLOAD_TIMEOUT='--max-time $(or $2,$1)'
 else

From 1c8e6543a4614ab6ee6c75efb21db1d35f234e47 Mon Sep 17 00:00:00 2001
From: Miguel Angel Nubla <miguelangel.nubla@gmail.com>
Date: Thu, 1 Jun 2023 02:27:45 +0200
Subject: [PATCH 2/2] Enable follow redirect for wget downloader in
 configure.ac

---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 0a542e7d88c..bd8add5e4dd 100644
--- a/configure.ac
+++ b/configure.ac
@@ -292,7 +292,7 @@ DOWNLOAD=":"
 DOWNLOAD_TO_STDOUT=
 DOWNLOAD_TIMEOUT=
 if test "x${WGET}" != "x:"; then
-  DOWNLOAD=${WGET}
+  DOWNLOAD="${WGET} -L"
   DOWNLOAD_TO_STDOUT="${WGET} -q -O-"
   DOWNLOAD_TIMEOUT='--timeout=$1'
 elif test "x${CURL}" != "x:"; then
