diff --git a/rtsp_sip_links.sh b/rtsp_sip_links.sh
index e1f59db..e15cacc 100755
--- a/rtsp_sip_links.sh
+++ b/rtsp_sip_links.sh
@@ -1,23 +1,22 @@
 #!/bin/bash
 cd include
-sudo ln -s ../third-party/pjproject/source/pjlib/include/pj pj
-sudo ln -s ../third-party/pjproject/source/pjlib/include/pjlib.h pjlib.h
-sudo ln -s ../third-party/pjproject/source/pjlib-util/include/pjlib-util pjlib-util
-sudo ln -s ../third-party/pjproject/source/pjlib-util/include/pjlib-util.h pjlib-util.h
-sudo ln -s ../third-party/pjproject/source/pjmedia/include/pjmedia pjmedia
-sudo ln -s ../third-party/pjproject/source/pjmedia/include/pjmedia-audiodev pjmedia-audiodev
-sudo ln -s ../third-party/pjproject/source/pjmedia/include/pjmedia-codec pjmedia-codec
-sudo ln -s ../third-party/pjproject/source/pjmedia/include/pjmedia-codec.h pjmedia-codec.h
-sudo ln -s ../third-party/pjproject/source/pjmedia/include/pjmedia.h pjmedia.h
-sudo ln -s ../third-party/pjproject/source/pjmedia/include/pjmedia-videodev pjmedia-videodev
-sudo ln -s ../third-party/pjproject/source/pjmedia/include/pjmedia_videodev.h pjmedia_videodev.h
-sudo ln -s ../third-party/pjproject/source/pjnath/include/pjnath pjnath
-sudo ln -s ../third-party/pjproject/source/pjnath/include/pjnath.h pjnath.h
-sudo ln -s ../third-party/pjproject/source/pjsip/include/pjsip pjsip
-sudo ln -s ../third-party/pjproject/source/pjsip/include/pjsip.h pjsip.h
-sudo ln -s ../third-party/pjproject/source/pjsip/include/pjsip-simple pjsip-simple
-sudo ln -s ../third-party/pjproject/source/pjsip/include/pjsip_simple.h pjsip_simple.h
-sudo ln -s ../third-party/pjproject/source/pjsip/include/pjsip-ua pjsip-ua
-sudo ln -s ../third-party/pjproject/source/pjsip/include/pjsip_ua.h pjsip_ua.h
-sudo ln -s ../third-party/pjproject/source/pjsip/include/pjsua-lib pjsua-lib
-
+ln -s ../third-party/pjproject/source/pjlib/include/pj pj
+ln -s ../third-party/pjproject/source/pjlib/include/pjlib.h pjlib.h
+ln -s ../third-party/pjproject/source/pjlib-util/include/pjlib-util pjlib-util
+ln -s ../third-party/pjproject/source/pjlib-util/include/pjlib-util.h pjlib-util.h
+ln -s ../third-party/pjproject/source/pjmedia/include/pjmedia pjmedia
+ln -s ../third-party/pjproject/source/pjmedia/include/pjmedia-audiodev pjmedia-audiodev
+ln -s ../third-party/pjproject/source/pjmedia/include/pjmedia-codec pjmedia-codec
+ln -s ../third-party/pjproject/source/pjmedia/include/pjmedia-codec.h pjmedia-codec.h
+ln -s ../third-party/pjproject/source/pjmedia/include/pjmedia.h pjmedia.h
+ln -s ../third-party/pjproject/source/pjmedia/include/pjmedia-videodev pjmedia-videodev
+ln -s ../third-party/pjproject/source/pjmedia/include/pjmedia_videodev.h pjmedia_videodev.h
+ln -s ../third-party/pjproject/source/pjnath/include/pjnath pjnath
+ln -s ../third-party/pjproject/source/pjnath/include/pjnath.h pjnath.h
+ln -s ../third-party/pjproject/source/pjsip/include/pjsip pjsip
+ln -s ../third-party/pjproject/source/pjsip/include/pjsip.h pjsip.h
+ln -s ../third-party/pjproject/source/pjsip/include/pjsip-simple pjsip-simple
+ln -s ../third-party/pjproject/source/pjsip/include/pjsip_simple.h pjsip_simple.h
+ln -s ../third-party/pjproject/source/pjsip/include/pjsip-ua pjsip-ua
+ln -s ../third-party/pjproject/source/pjsip/include/pjsip_ua.h pjsip_ua.h
+ln -s ../third-party/pjproject/source/pjsip/include/pjsua-lib pjsua-lib
